ops/docker/Dockerfile.monorepo

```diff
@@ -45,17 +46,21 @@ WORKDIR /optimism
 # code changes
 COPY --from=builder /optimism/node_modules ./node_modules
 COPY --from=builder /optimism/packages ./packages
+COPY --from=builder /optimism/integration-tests ./integration-tests
 
 # the following steps are cheap
 COPY *.json yarn.lock ./
 # copy over the source
 COPY ./packages ./packages
+COPY ./integration-tests ./integration-tests
 # copy over solc to save time building (35+ seconds vs not doing this step)
 COPY --from=downloader solc /root/.cache/hardhat-nodejs/compilers/linux-amd64/solc-linux-amd64-v0.7.6+commit.7338295f
-COPY --from=downloader ovm-solc /root/.cache/hardhat-nodejs/compilers/ovm/0.7.6.js
+COPY --from=downloader ovm-solc /root/.cache/hardhat-nodejs/compilers/ovm/0.7.6-allow_kall_2.js
```

Comment:
Can we copy the `compilers` directory over so that we don't need to hard code versions here?